include/IFileSystem.h

```diff
@@ -260,7 +260,7 @@ class IFileSystem : public virtual core::IReferenceCounted
 #else
 		//in one case, IRR_CORE_UNIQUE_STRING_LITERAL_TYPE value returns an incorrect string
 		//and specializedshader_fragment becomes -> specializedshader_fragmenttt
-		auto path = (std::string(__IRR_ROOT_DIRECTORY__) + "/include/" + std::string(StringUniqueType::value));	
+		auto path = (std::string(__IRR_INSTALL_DIR__) + "/include/" + std::string(StringUniqueType::value));

```

Comment:
I need to see if this will work

Reply:
we should solve it differently

Reply:
I am unsure how to pass the reference of SIrrlichtCreationParameter to IFileSystem